README.md

```diff
@@ -16,7 +16,9 @@ you plan to use new (separate) VPC.
 * `namespace`: (Required) Namespace associated with these resources
 * `vpc_id`: (Required) AWS Virtual Private Cloud ID.
 * `vpc_default_route_table_id`: A default route table for public subnets. Provides access to Internet. If not set here - will be created.
+* `create_network_acl`: The new set of ACLs that will be associated with these resources
```

Comment:
I think we should always create it unless the user specifies a `public_network_acl` or `private_network_acl`